.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2
 jobs:
 
-  api-build-test-deploy:
```

Comment:
Breaking up the build/test from the deploy step means that we wind up checking out and building the code twice for each deployment... correct? How much does that slow things down?

Reply:
Here's a normal run for a master merge:
https://circleci.com/gh/all-of-us/workbench/9736

Here's a normal run for a topic branch:
https://circleci.com/gh/all-of-us/workbench/9700

The overhead of running a job with our build image seems to be ~1m. Code checkout is negligible. Builds are cached, but it's difficult to say how much work is being redone on each run.

- There should be no change here on open PR circle runs.
- Master merges will be slightly slower (I expect 1-2m slower). The improvement on master merge is that we only push if all tests pass. Previously we might push the UI despite broken API tests, or we might push the API despite broken api-local tests, for example.

I will make some temporary changes to this PR to simulate a master merge so we can get real numbers.